src/game/gui/menu.c

```diff
@@ -411,6 +416,11 @@ void menu_set_padding(component *c, int padding) {
     m->padding = padding;
 }
 
+void menu_invalidate_help_text_cache(component *c) {
```

Comment:
Does anything use this?

Reply:
game_menu_return should be calling it.

Reply:
Maybe I goofed the rebase?

Reply:
Could also be I missed it in the original PROTO submit.